### PR TITLE
docker added

### PR DESCRIPTION
**Pull request by sohailg**

### CODE REVIEW

File: src/list.json [hunk 1]
```diff
@@ -54,6 +54,14 @@
         "description": "Create two PRs in any of the qualifying Wichita repos.",
         "no_of_prs": 2,
         "link": "https://devict-hacktoberfest.herokuapp.com/"
+      },
+	  {
+        "organization": "Docker",
+        "org_url": "https://www.docker.com/",
+        "tags": ["T-shirt","Hoodie"],
```
Comment (monizb) [inline src/list.json:61]: T-shirt ----> Tshirt

File: src/list.json [hunk 2]
```diff
@@ -54,6 +54,14 @@
         "description": "Create two PRs in any of the qualifying Wichita repos.",
         "no_of_prs": 2,
         "link": "https://devict-hacktoberfest.herokuapp.com/"
+      },
+	  {
```
Comment (monizb) [inline src/list.json:58]: Formatting --> Please format the json

**Review by monizb — CHANGES_REQUESTED** [review]

File: src/list.json (hunk 1 above)
Comment (sohailg) [reply src/list.json]: changes done

**Comment by sohailg** [comment]
I request you to look into 
formatting done [30c1a50](https://github.com/monizb/hacktoberfest-swag-list/pull/149/commits/30c1a5083a9b7c1792999a9ea3bc08e590289fde)

**Pull request merged**